scripts/generate_bge_exome_calling_regions.sh

```diff
@@ -32,7 +32,7 @@ rm tmp.interval_list
 # =============== mirbase.v22.interval_list ================
 # ==========================================================
 
-wget -nc -O mirbase.v22.gff https://www.mirbase.org/ftp/22/genomes/hsa.gff3
+wget -nc --no-check-certificate -O mirbase.v22.gff https://www.mirbase.org/ftp/22/genomes/hsa.gff3
```

Comment:
was the `--no-check-certificate` necessary?  if it's not needed, it does make me a little nervous, tbh.

Reply:
I think I only needed this due to the environment I was running in. I reran on my laptop and didn't need this argument and then double checked that the outputs matched and everything looked good.